samples/csharp/getting-started/Clustering_CustomerSegmentation/README.md

```diff
@@ -61,6 +61,52 @@ To solve this problem, first we will build an ML model. Then we will train the m
 
 ![](https://raw.githubusercontent.com/dotnet/machinelearning-samples/features/samples-new-api/samples/csharp/getting-started/shared_content/modelpipeline.png)
 
+#####  Calculate Best K
+
+There is only one problem. How do we know how many clusters our data should have? This is a real problem for which there is no perfect solution. In this sample we used the Elbow method [paper](http://web.stanford.edu/~hastie/Papers/gap.pdf).
+
+![elbow plot](./CustomerSegmentation.Train/assets/outputs/elbow.svg)
+
+As you can see, at k=5 the graph begins to flatten significantly. This point where the graph starts to smooth out is the prophesied “elbow” for which we have been looking.
```

Comment:
I'm uncertain I'm see an elbow here

Reply:
@justinormont - Could you be more specific on this topic? Is this PR worth to be taken in order to suggest the best number for K? - If not, what other approach would you suggest?

Reply:
In the SVG, I do not see a clear elbow in the graph:
https://github.com/dotnet/machinelearning-samples/blob/d1517aac36b1c3071e2b07972e84ef13a5a31e01/samples/csharp/getting-started/Clustering_CustomerSegmentation/CustomerSegmentation.Train/assets/outputs/elbow.svg



I was expecting something more like: 
https://datascience.stackexchange.com/questions/6508/k-means-incoherent-behaviour-choosing-k-with-elbow-method-bic-variance-explain